### ISSUE TITLE
Question about Evaluating

### ISSUE THREAD

**Issue by zeratul233** [SuHoHan95/VISOLO#5]
How can I obtain the FPS and AP data for the Neural Network on the validation set?
